cogip/widgets/lidarview.py

```diff
@@ -23,41 +23,48 @@ class LidarView(QtWidgets.QWidget):
     """
     min_distance: int = 100
     max_distance: int = 5000
-    max_intensity: int = 12000
+    max_intensity: int = 1024
```

Comment:
Is it the max intensity you define or the max intensity the LIDAR is capable of ?
maybe need some few small comments here.

Reply:
Comment added

Reply:
Actually it is both: max intensity of the Lidar and max intensity to display.